buildtest/schemas/spack.schema.json

```diff
@@ -118,6 +118,21 @@
             }
           }
         },
+        "deactivate": {
+          "additionalProperties": false,
+          "type": "object",
+          "description": "Deactivate a spack environment prior to activating it via ``spack env deactivate``",
+          "properties": {
+            "name": {
+              "type": "boolean",
+              "description": "Deactivate any active spack environment before activating the environment. If set to ``True`` we will run ``spack env deactivate``."
+            },
+            "options": {
+              "type": "string",
+              "description": "Pass options to ``spack env deactivate`` command"
+            }
```

Comment:
please remove `options` there is no need for this when mapping options to  `spack env deactivate`. Right now none of the options are useful